### PR TITLE
Enhance search box

### PR DESCRIPTION
**Pull request by kulendu**
## Description
- List out your changes : 
   There are overall two UI changes I have made for the search box, they are as follows :
   1.  Increasing the **border-radius** of the search box, which in result will look more minimal and goes well with the theme.
   2. Secondly, making the search box move towards the right side of the top bar, close to the other tab section.
- Link the Issue which this resolves: #168 

## Affected Dependencies : 
No dependencies were affected during my code changes.

## How has this been tested?
This has been tested by using **flake8-django**. This is a plugin used to detect bad practices on Django projects. And also we  have used a file for checking the test cases (**manage.py test**) and successfully it has passed all the mentioned test cases.

Annotation 2021-04-02 232717

## GIF Before
Annotation 2021-04-01 184233

## GIF After
Annotation 2021-04-01 184416

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/Praful932/Kitabe/blob/master/CONTRIBUTING.md).
- [x] My changes do not edit/add any unrequired files.
- [x] My changes are covered by tests.

### CODE REVIEW

**Comment by kulendu** [comment]
@Praful932 please have a look at this PR 😄

File: static/mainapp/css/layout.css [hunk 1]
```diff
@@ -45,11 +45,11 @@ body {
 }
 
 #bookName {
-  border-radius: 10px;
+  border-radius: 50px;
 }
 
 #searchdropdown {
-  margin-left: 50%;
+  margin-left: 57%;
```
Comment (maneprajakta) [inline static/mainapp/css/layout.css:52]: let it be 50%

**Review by maneprajakta — CHANGES_REQUESTED** [review]
- Margin left 57% dosen't work when user signs in 
Screenshot from 2021-04-04 11-01-34

File: static/mainapp/css/layout.css (hunk 1 above)
Comment (kulendu) [reply static/mainapp/css/layout.css:52]: But @maneprajakta when I signed in my margin looked fine.

Annotation 2021-04-04 111822 

Let's see what others say! and if its a problem for others, then surely I would keep it as default (50%)

**Review by Praful932 — CHANGES_REQUESTED** [review]
@kulendu The live deployment isn't as it seems from your attached ss
Before log in
image

After log in
image

**Comment by kulendu** [comment]
> @kulendu The live deployment isn't as it seems from your attached ss
> Before log in
> image
> 
> After log in
> image

No actually I made the changes in the Home page itself, and also reviewd it there. If you want I can sent yot the sreenshot for the **Log in** page after and before

**Comment by Praful932** [comment]
That's fine but even if you do, that won't change how it looks on the deployment
@maneprajakta Does this have to do with the resolution of the display like we faced before?

**Comment by kulendu** [comment]
So @Praful932 if you want me to revert my changes back it won't be a problem. I thought to improve the top bar a bit, but that's fine 😄

**Comment by Praful932** [comment]
> So @Praful932 if you want me to revert my changes back it won't be a problem. I thought to improve the top bar a bit, but that's fine 😄

let's confirm if this is the resolution issue first, could you check [here ](https://kitabe-pipeline-pr-174.herokuapp.com/)after log in?

**Comment by kulendu** [comment]
> > So @Praful932 if you want me to revert my changes back it won't be a problem. I thought to improve the top bar a bit, but that's fine 😄
> 
> let's confirm if this is the resolution issue first, could you check [here ](https://kitabe-pipeline-pr-174.herokuapp.com/)after log in?

@Praful932 the resolution is fine.
Annotation 2021-04-04 234602

**Comment by Praful932** [comment]
@kulendu  Is your zoom level set to 100%?

**Comment by kulendu** [comment]
Yes.

**Comment by maneprajakta** [comment]
> Yes.

what's your screen resolution?

**Comment by kulendu** [comment]
> > Yes.
> 
> what's your screen resolution?

1920 X 1080

**Comment by maneprajakta** [comment]
@kulendu resolution close to 1360x768 would work.

**Comment by kulendu** [comment]
> @kulendu resolution close to 1360x768 would work.

@Praful932 and @maneprajakta can you please be specific to me the issue we are facing?

**Comment by Praful932** [comment]
> > @kulendu resolution close to 1360x768 would work.
> 
> @Praful932 and @maneprajakta can you please be specific to me the issue we are facing?

Check the page on `1366 * 768` resolution, That's the general resolution the website is build on

**Comment by kulendu** [comment]
@Praful932 This is what its showing on 1366 X 768 : 

Annotation 2021-04-06 220426

**Comment by Praful932** [comment]
> @Praful932 This is what its showing on 1366 X 768 :
> 
> Annotation 2021-04-06 220426

Exactly.

**Comment by kulendu** [comment]
> > @Praful932 This is what its showing on 1366 X 768 :
> > Annotation 2021-04-06 220426
> 
> Exactly.

So what should I do now? Should I close this PR? or another thing could be done that we can keep the **margin-left** as it is and can make the changes in the **border-radius** as I proposed .  That will look  better I guess

**Comment by maneprajakta** [comment]
> So what should I do now? Should I close this PR? or another thing could be done that we can keep the **margin-left** as it is and can make the changes in the **border-radius** as I proposed . That will look better I guess

Yep keep **margin-left** as it is just change the border-radius.

**Comment by kulendu** [comment]
Ok

**Review by kulendu — COMMENTED** [review]
@Praful932 and @maneprajakta please have a look now. I have changed the **margin-left** as the default value and only changed the **border-radius**

**Review by maneprajakta — APPROVED** [review]

**Comment by kulendu** [comment]
@maneprajakta since you reviewed it. So @Praful932 is this PR now in the stage of being merged?

**Comment by Praful932** [comment]
> @maneprajakta since you reviewed it. So @Praful932 is this PR now in the stage of being merged?

Checking

**Review by Praful932 — APPROVED** [review]
LGTM

**Pull request merged**